### PR TITLE
Testimonials Section

### PR DESCRIPTION
**Pull request by tsukuros**
This is the first iteration of Testimonials carousel.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -129,10 +129,76 @@ <h3 class="mb-4 section-subtitle text-bold text-proxima">What are my accomplishm
...
+            <span class="sr-only">Previous</span>
+          </a>
+          <a class="carousel-control-next" href="#carouselExampleIndicators" role="button" data-slide="next">
+            <span class="carousel-control-next-icon" aria-hidden="true">
+              <img class="img-fluid" src="images/arrow-left.svg" alt="Arrow Right Icon" />
+            </span>
+            <span class="sr-only">Next</span>
+          </a>
+        </div>
+
```
Comment (tsukuros) [inline index.html:195]: Remove empty line.

File: index.html [hunk 2]
```diff
@@ -129,10 +129,76 @@ <h3 class="mb-4 section-subtitle text-bold text-proxima">What are my accomplishm
...
+          <ol class="carousel-indicators">
+            <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active">
+              <img
+                class="d-block img-fluid rounded-circle"
+                src="images/person-1.jpg"
+                srcset="images/person-1@2x.jpg 2x"
+                alt="Person 1"
+              />
+            </li>
+            <li data-target="#carouselExampleIndicators" data-slide-to="1">
```
Comment (tsukuros) [inline index.html]: Use specific to this project name for the carousel. `carouselExampleIndicators ` doesn't make any sense.
I recommend:
```
data-target="#testimonials-carousel"
```

**Review by tsukuros — COMMENTED** [review]
@melnichenkod and @bmushtuk please review my code.

**Comment by tsukuros** [comment]
@tsukuros Please remove the jumping behavior when clicking indicators. See demonstration below:

jumpy-indicators

File: index.html [hunk 3]
```diff
@@ -129,10 +129,76 @@ <h3 class="mb-4 section-subtitle text-bold text-proxima">What are my accomplishm
...
+                  <div>
+                    <p class="person-name text-proxima text-20 mb-1">Vlada Boiko</p>
+                    <p class="person-title text-proxima mb-0">UI/UX Designer</p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+
+          <a class="carousel-control-prev" href="#carouselExampleIndicators" role="button" data-slide="prev">
```
Comment (bmushtuk) [inline index.html]: Change `href="#carouselExampleIndicators"` so that it corresponds to the specific to this project id

**Review by ghost — APPROVED** [review]

**Review by bmushtuk — APPROVED** [review]

**Pull request merged**